Octokit.Tests.Integration/Clients/ReleasesClientTests.cs

```diff
@@ -75,6 +75,83 @@ public void Dispose()
         }
     }
 
+    public class TheGetAllMethod
+    {
+        readonly IReleasesClient _releaseClient;
+        const string owner = "octokit";
+        const string name = "octokit.net";
+
+        public TheGetAllMethod()
+        {
+            var github = Helper.GetAuthenticatedClient();
+            _releaseClient = github.Repository.Release;
+        }
+
+        [IntegrationTest]
+        public async Task ReturnsReleases()
+        {
+            var releases = await _releaseClient.GetAll(owner, name);
+
+            Assert.NotEmpty(releases);
+        }
+
+        [IntegrationTest]
+        public async Task ReturnsCorrectCountOfReleasesWithoutStart()
+        {
+            var options = new ApiOptions
+            {
+                PageSize = 5,
+                PageCount = 1
+            };
+
+            var releases = await _releaseClient.GetAll(owner, name, options);
+
+            Assert.Equal(5, releases.Count);
+        }
+
+        [IntegrationTest]
+        public async Task ReturnsCorrectCountOfReleasesWithStart()
+        {
+            var options = new ApiOptions
+            {
+                PageSize = 5,
+                PageCount = 1,
+                StartPage = 2
+            };
+
+            var releases = await _releaseClient.GetAll(owner, name, options);
+
+            Assert.Equal(5, releases.Count);
+        }
+
+        [IntegrationTest]
+        public async Task ReturnsDistinctResultsBasedOnStartPage()
+        {
+            var startOptions = new ApiOptions
+            {
+                PageSize = 5,
+                PageCount = 1
+            };
+
+            var firstPage = await _releaseClient.GetAll(owner, name, startOptions);
+
+            var skipStartOptions = new ApiOptions
+            {
+                PageSize = 5,
+                PageCount = 1,
+                StartPage = 2
+            };
+
+            var secondPage = await _releaseClient.GetAll(owner, name, skipStartOptions);
+
+            Assert.NotEqual(firstPage[0].Id, secondPage[0].Id);
```

Comment:
Ive hit this quite a few times when writing tests as well.  Would be great to be able to do ```Equality``` on the response models (and request models for that matter).  If you start a separate discussion I'd be interested to see what the plans are, however for this PR I'd say the test above is fine